crates/air_r_formatter/src/r/auxiliary/root.rs

```diff
@@ -1,10 +1,27 @@
 use crate::prelude::*;
 use air_r_syntax::RRoot;
 use air_r_syntax::RRootFields;
+use air_r_syntax::RSyntaxNode;
 use biome_formatter::write;
+use biome_formatter::CstFormatContext;
+use biome_rowan::Direction;
+use biome_rowan::SyntaxElement;
+use comments::parse_comment_directive;
+use comments::Directive;
+use comments::FormatDirective;
 
 #[derive(Debug, Clone, Default)]
 pub(crate) struct FormatRRoot;
+
+impl FormatRRoot {
+    fn is_suppression_comment(text: &str) -> bool {
+        matches!(
+            parse_comment_directive(text),
+            Some(Directive::Format(FormatDirective::SkipFile))
+        )
+    }
+}
```

Comment:
I think I would just inline this at this point?

Reply:
It did use to be more complex

Reply:
Removed it then had to add it back